.github/workflows/test-pull-request-dotnet-core.yml

```
@@ -14,6 +14,8 @@ jobs:
     steps:
     - name: Setup Go
       uses: actions/setup-go@v2.2.0
+      with:
+        go-version: 1.17
```

```suggestion
        go-version: 1.17.x
```

I think you want to also specify that the patch number can float given that `1.17` is considered to be a specific version.

---

Good call.